apps/ledger-live-desktop/src/newArch/features/ModularDrawer/components/Search/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Icons } from "@ledgerhq/react-ui/index";
+import { Input } from "@ledgerhq/react-ui/pre-ldls";
+import { SearchProps, useSearch } from "./useSearch";
+
+export const SearchItem = ({ setItemsToDisplay, source, flow, items }: SearchProps) => {
+  const { t } = useTranslation();
+
+  const { searchQuery, handleSearch } = useSearch({
+    setItemsToDisplay,
+    items,
+    source,
+    flow,
+  });
+
+  return (
+    <Input
```

Comment:
Please use the Search component from the `@ledgerhq/react-ui/pre-ldls` https://react-ui-storybook.vercel.app/?path=/story/preldls-components-search--default (the main difference is the `onDebouncedChange` which is not usefull rn but it should be later).